docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -106,5 +106,14 @@ Analysis takes minutes to hours to run depending on the size of the application
 ====
 {ProductShortName} relies on Kubernetes scheduling capabilities to determine how many analyzer instances are created based on cluster capacity. If several applications are selected for analysis, by default, only one analyzer can be provisioned at a time.  With more cluster capacity, more analysis processes can be executed in parallel.
 ====
-
 . When analysis is complete, you can click the *Report* link to see the results of the analysis.
+
+[id="viewing-analysis-details_{context}"]
+== Viewing analysis details
+
+You can view the details of an analysis by clicking the options menu ({kebab}) and selecting *Analysis details*. The details are displayed in the *Analysis details for customers* window. You can choose either YAML or JSON format.
```

Comment:
@sbeskin-redhat  If you look at the rest of the MTA documentation, you will notice that the format for options menu kebab is **Options menu {kebab}**. Please make this change. And in future, please ensure you are consistent. If you are unsure about format used, check the existing documentation.

Reply:
@sbeskin-redhat - 
![image](https://github.com/windup/windup-documentation/assets/106804821/2c067a08-9e9d-4271-ab59-69e28f8c3c08)
![image](https://github.com/windup/windup-documentation/assets/106804821/3af4c488-b550-4f1f-a841-b52e206c15e0)

Reply:
This menu does not have any visible name in the UI.

Reply:
@sbeskin-redhat  if there is no title when you hover over the menu kebab, that is a UI issue, and you should report it. There might not be any text to be picked up by a Screen Reader, which is an accessibility issue.